### DISCUSSION TITLE
Which Devil Fruit in OnePiece do you want?(欲しい悪魔の実を教えてください)

### DISCUSSION THREAD

**Discussion by ShuSakamoto** [NAIST-SE/AttractingContributors#424] [General]
Tell me which Devil Fruit in OnePiece you want! I want Love-Love fruit very much.
ワンピースに登場する悪魔の実の中で、あなたが欲しいものを教えてください。僕はメロメロの実がとても欲しいです。

Link: https://github.com/ShuSakamoto/AttractingContributers-DevilFruit

**Comment by aru1702** [comment]
My favorite is Hie hie no mi, which Aokiji freezing fruit.
How about you?

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407
